chapters/VK_QNX_external_memory_screen_buffer/qnx_screen_buffer.adoc

```diff
@@ -0,0 +1,111 @@
+// Copyright 2023 QNX Software Systems
+//
+// SPDX-License-Identifier: CC-BY-4.0
+
+[[memory-external-qnx-screen-buffer]]
+==== QNX Screen Buffer
+
+The QNX SDP defines code:_screen_buffer objects, which represent
+a buffer that the QNX Screen graphics subsystem can use directly
+in its windowing system APIs. More specifically, a Screen buffer is an area of memory that
+stores pixel data. It can be attached to Screen windows, streams, or pixmaps.
+These QNX Screen buffer objects may: be imported into
+slink:VkDeviceMemory objects for access via Vulkan.
+An slink:VkImage or slink:VkBuffer can: be bound to the imported
+slink:VkDeviceMemory object if it is created with
+ename:VK_EXTERNAL_MEMORY_HANDLE_TYPE_SCREEN_BUFFER_BIT_QNX.
+
+--
+code:struct code:_screen_buffer is strongly typed, so naming the handle type is
```

Comment:
> from link http://www.qnx.com/developers/docs/7.1/index.html#com.qnx.doc.screen/topic/screen_buffer_t.html screen_buffer_t is the handle of _screen_buffer, since _screen_buffer is an opaque data structure, it can not be used directly. so I think screen_buffer_t should be used in interface.

We cannot use opaque types directly, because in many cases, including validation layer the generated code is trying to get sizeof() of such incomplete types. For struct pointers we can avoid dereference of such types (in most cases).